src/styles/stars.css

```diff
@@ -0,0 +1,26 @@
+/* #region Stars */
```

Comment:
you have already separated blocks into files. there is no need in comments, remove it